android/java/org/chromium/chrome/browser/crypto_wallet/adapters/WalletCoinAdapter.java

```diff
@@ -139,6 +140,14 @@ public void onBindViewHolder(@NonNull ViewHolder holder, int position) {
             }
         });
 
+        holder.itemView.setClickable(true);
+        if (walletListItemType == Utils.TRANSACTION_ITEM
```

Comment:
`walletListItemType == Utils.TRANSACTION_ITEM` is redundant, please move it inside the existing block

Reply:
I did this because the existing block `if (walletListItemType == Utils.TRANSACTION_ITEM) {` before the click listener is set, so if I do that, ripple effect is shown when we tap on rejected item, which does not feel disabled.

Reply:
simply move the code above or vise-versa